pfcpiface/main.go

```diff
@@ -19,13 +19,20 @@ const (
 	modeSim = "sim"
 )
 
+//grpc channel state
+const (
+	Ready = 2 //grpc channel state Ready
+)
+
 var (
-	bessIP     = flag.String("bess", "localhost:10514", "BESS IP/port combo")
-	configPath = flag.String("config", "upf.json", "path to upf config")
-	httpAddr   = flag.String("http", "0.0.0.0:8080", "http IP/port combo")
-	simulate   = flag.String("simulate", "", "create|delete simulated sessions")
-	pfcpsim    = flag.Bool("pfcpsim", false, "simulate PFCP")
-	n4SrcIPStr = flag.String("n4SrcIPStr", "", "N4Interface IP")
+	bessIP          = flag.String("bess", "localhost:10514", "BESS IP/port combo")
+	configPath      = flag.String("config", "upf.json", "path to upf config")
+	httpAddr        = flag.String("http", "0.0.0.0:8080", "http IP/port combo")
+	simulate        = flag.String("simulate", "", "create|delete simulated sessions")
+	pfcpsim         = flag.Bool("pfcpsim", false, "simulate PFCP")
+	n4SrcIPStr      = flag.String("n4SrcIPStr", "", "N4Interface IP")
+	p4RtcServerIP   = flag.String("p4RtcServerIP", "", "P4 Server ip")
+	p4RtcServerPort = flag.String("p4RtcServerPort", "", "P4 Server port")
```

Comment:
can we group bess/p4 flags and move into their files. Is that possible?

Reply:
done